gcloud/bigtable/row.py

```diff
@@ -257,6 +257,78 @@ def to_pb(self):
         return data_pb2.RowFilter(value_regex_filter=self.regex)
 
 
+class ValueRangeFilter(RowFilter):
+    """A range of values to restrict to in a row filter.
+
+    Will only match cells that have values in this range.
+
+    Both the start and end value can be included or excluded in the range.
+    By default, we include them both, but this can be changed with optional
+    flags.
+
+    :type start_value: bytes
+    :param start_value: The start of the range of values. If no value is
+                        used, it is interpreted as the empty string
+                        (inclusive) by the backend.
+
+    :type end_value: bytes
+    :param end_value: The end of the range of values. If no value is used, it
+                      is interpreted as the infinite string (exclusive) by the
+                      backend.
+
+    :type inclusive_start: bool
+    :param inclusive_start: Boolean indicating if the start value should be
+                            included in the range (or excluded). Defaults
+                            to :data:`True`.
+
+    :type inclusive_end: bool
+    :param inclusive_end: Boolean indicating if the end value should be
+                          included in the range (or excluded). Defaults
+                          to :data:`True`.
+    """
+
+    def __init__(self, start_value=None, end_value=None,
+                 inclusive_start=True, inclusive_end=True):
+        self.start_value = start_value
+        self.end_value = end_value
+        self.inclusive_start = inclusive_start
+        self.inclusive_end = inclusive_end
```

Comment:
Or use a sentinel as the default, and convert it to `True` after checking for an explicitly passed value.